[docs] GettingStarted: Add a warning about spaces in the absolute path (#66456)
swiftlang/swift: docs/HowToGuides/GettingStarted.md

   mkdir swift-project
   cd swift-project
   ```
   
> **Warning**  

> > Make sure the absolute path to your `swift-project` folder **does not** contain spaces, 

Reviewer: Could we stick to "directory" for consistency?

>     since that might cause build issues on the next steps.

Reviewer: Does this sound better?

```suggestion
    since that might cause issues during the build step.
```

>     since that might cause build issues on the next steps.

Reviewer: The warning needs to be indented (see how it renders on GitHub).

>         since that might cause build issues during the build step.

Reviewer: ```suggestion
        since that might cause issues during the build step.
```
